tools/integration_tests/run_e2e_tests.sh

```diff
@@ -22,6 +22,9 @@ RUN_E2E_TESTS_ON_PACKAGE=$1
 # By default, this script runs all the integration tests.
 SKIP_NON_ESSENTIAL_TESTS_ON_PACKAGE=$2
 
+# The location (region) where bucket should be created.
+BUCKET_LOCATION=$3
```

Comment:
Should we keep default value something if user does not pass any? It will be helpful when someone run this script locally.

Reply:
I would rather enforce this because for them there are chances that the person executing this script will also forget to pass location and their VM location may be different from default bucket location, increasing the time to run.

Reply:
Hmm...okay then just suggestion, should we put check that $3 is not passed then echo "Pass bucket location same as your VM location". Currently it will break at the time of creating bucket.

Reply:
I have added a check to ensure all the three flags are passed for now because we need to pass all three flags (we can't simply pass bucket location without passing others). If required in future, we will move to take arguments like https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/read_cache/run_read_cache_fio_workload.sh

Reply:
probable better to put an error log if `$#` < 3.